.github/workflows/devbuild.yml

```diff
@@ -258,7 +269,18 @@ jobs:
             # GitHub Action macOS 13 has a problem with python3
             brew list -1 | grep python | while read formula; do brew unlink $formula; brew link --overwrite $formula; done;
           fi
-          brew install qt6
+          # brew install qt6
+
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v3
+        with:
+          aqtversion: '==3.1.*'
```

Comment:
`==3.1` should be fine?

Reply:
this is a version specifier using in `pip install aqtinstall==<version specifier>`
https://github.com/jurplel/install-qt-action/blob/master/action/src/main.ts#L354

the version is just fellowing the current default.
https://github.com/jurplel/install-qt-action/tree/master?tab=readme-ov-file#aqtversion

Reply:
True, just realized it follows pip's rule.
Thanks for the explanation.